core/src/components/radio-group/radio-group.scss

```diff
@@ -0,0 +1,3 @@
+ion-radio-group {
+  flex: 1;
+}
```

Comment:
What does this do?

Reply:
This causes the radio group to fill the remaining space of the container, alternatively we could use `width: 100%` here, it will have the same visual effect in this instance.

e.g.:

Without filling remaining space:

![localhost_3333_src_components_radio_test_item (1)](https://user-images.githubusercontent.com/13732623/210864945-92bd0d38-2ed7-4933-9c1a-074cd8ae2270.png)

Filling remaining space:

![localhost_3333_src_components_radio_test_item](https://user-images.githubusercontent.com/13732623/210864951-be963751-45cd-4f0a-938d-4f76912ddf1b.png)

This ties to this comment: https://github.com/ionic-team/ionic-framework/pull/26499#issuecomment-1360393586.